docs/DGDiagramsCreator/UI.puml

```diff
@@ -0,0 +1,17 @@
+@startuml
+!include Style.puml
+title UI Component Diagram
+actor User
+node SecureNUS
```

Comment:
Maybe represent the classes using the standard representation adopted by CS2113 instead of 3-dimensional boxes?